dll/win32/profmap/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+
+spec2def(profmap.dll profmap.spec ADD_IMPORTLIB)
+
+list(APPEND SOURCE
+    profmap_main.c
+    profmap.rc
+    ${CMAKE_CURRENT_BINARY_DIR}/profmap_stubs.c
+    ${CMAKE_CURRENT_BINARY_DIR}/profmap.def)
+
+add_library(profmap MODULE ${SOURCE})
+set_module_type(profmap win32dll)
+add_importlibs(profmap userenv netapi32 user32 advapi32 msvcrt kernel32 ntdll)
```

Comment:
Considering these are only stubs, do we need `userenv`, `netapi32`, `user32`, `advapi32` ?

Reply:
I just did it same as in MS implementation (according to the advanced information about this dll: https://xpdll.nirsoft.net/profmap_dll.html). Ideally they should be imported, but if that importlibs currently are not used by the profmap stubs, then not necessarily to do this right now. Thanks.

Reply:
Fixed.